docs/cookbooks/creating_a_derived_dataset.md

```diff
@@ -106,11 +106,19 @@ BigQuery-ETL has some facilities in it to automatically add your query to [telem
 Before scheduling your query, you'll need to find an [Airflow DAG](https://airflow.apache.org/docs/apache-airflow/stable/concepts.html#dags) to run it off of. In some cases, one may already exist that makes sense to use for your dataset -- look in `dags.yaml` at the root or run `./bqetl dag info`. In this particular case, there's no DAG that really makes sense -- so we'll create a new one:
 
 ```bash
-./bqetl dag create bqetl_internal_tooling --schedule-interval "0 4 * * *" --owner wlachance@mozilla.com --description "This DAG schedules queries for populating queries related to Mozilla's internal developer tooling (e.g. mozregression)." --start-date 2020-06-01
+./bqetl dag create bqetl_internal_tooling --schedule-interval "0 4 * * *" --owner wlachance@mozilla.com --description "This DAG schedules queries for populating queries related to Mozilla's internal developer tooling (e.g. mozregression)." --start-date 2020-06-01 --tag impact/tier3
 ```
 
 Most of the options above should be self-explanatory. We use a schedule interval of "0 4 \* \* \*" (4am UTC daily) instead of "daily" (12am UTC daily) to make sure this isn't competing for slots with desktop and mobile product ETL.
 
+### Impact/priority tag
+We follow the same tiers as the ones defined for automation/treeherder. This is to maintain a level of consistency across different systems to ensure common language and understanding across teams. Valid tier tags include:
```

Comment:
This should probably be in a reference document rather than this long-form tutorial.

I would create a new document under `reference` like this one: https://mozilla.github.io/bigquery-etl/reference/scheduling/ (https://github.com/mozilla/bigquery-etl/blob/main/docs/reference/scheduling.md)

You can then (briefly) refer to it here.

Reply:
@wlach I added a doc: https://github.com/mozilla/bigquery-etl/blob/feat/airflow-tags/docs/reference/airflow_tags.md

Please let me know your thoughts.